editor/components/rich-text/format-toolbar/index.js

```diff
@@ -47,6 +47,11 @@ const FORMATTING_CONTROLS = [
 		shortcut: displayShortcut.primary( 'k' ),
 		format: 'link',
 	},
+	{
```

Comment:
Looks like we'll need something like #6642 so this can move to the block registration.